plugin/advisor/schema_review.go

```diff
@@ -220,7 +240,7 @@ func SchemaReviewCheck(ctx context.Context, statements string, policy *SchemaRev
 
 		advisorType, err := getAdvisorTypeByRule(rule.Type, context.DbType)
 		if err != nil {
-			log.Debug("not supported rule", zap.Error(err))
+			log.Println("not supported rule", zap.Error(err))
```

Comment:
The `github.com/pingcap/log` package is actually a wrapper based on `go.uber.org/zap`.
Maybe we can just use the system log and `log.Printf` here?

Reply:
LGTM